### PR TITLE
add systemd user service

### PR DESCRIPTION
**Pull request by kaikli**
I know it's quite easy to add a systemd service by myself, but why not add it to the package?

### CODE REVIEW

File: pkg/parcimonie.sh.user.service [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=parcimonie with config file ~/.config/parcimonie.sh.conf
+
+[Service]
+Type=simple
+EnvironmentFile=~/.config/parcimonie.sh.conf
```
Comment (EtiennePerot) [inline pkg/parcimonie.sh.user.service:6]: Are you sure that this works?
https://unix.stackexchange.com/a/534079 suggests that systemd doesn't expand `~` characters into the user's home directory.
According to https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers perhaps it should be `%h`?

**Review by EtiennePerot — CHANGES_REQUESTED** [review]
Thanks for contributing!

File: pkg/parcimonie.sh.user.service (hunk 1 above)
Comment (kaikli) [reply pkg/parcimonie.sh.user.service:6]: You are right. Thank you.

```suggestion
EnvironmentFile=-%E/parcimonie.sh.conf
```

The [`-`](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=) ignores missing files without raising an error and the [`%E`](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers) will be replaced with the configuration directory root (`$XDG_CONFIG_HOME` or `$HOME/.config`).

File: pkg/parcimonie.sh.user.service [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=parcimonie with config file ~/.config/parcimonie.sh.conf
+
+[Service]
+Type=simple
+EnvironmentFile=-%E/parcimonie.sh.conf
+ExecStart=/usr/bin/parcimonie.sh
+
+[Install]
+WantedBy=multi-user.target
```
Comment (kaikli) [inline pkg/parcimonie.sh.user.service:10]: mulit-user.target may not exist

**Review by EtiennePerot — APPROVED** [review]

**Comment by EtiennePerot** [comment]
Have you confirmed that it works now?

**Comment by kaikli** [comment]
> Have you confirmed that it works now?

Yes, I have tested it with my local arch linux.

**Comment by kaikli** [comment]
Can you update the PKGBUILD in AUR? Because now the new file is missing.

**Comment by EtiennePerot** [comment]
Done.

**Pull request merged**